src/applications/search/components/SearchDropDownComponent.js

```diff
@@ -0,0 +1,340 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import IconSearch from '@department-of-veterans-affairs/component-library/IconSearch';
+
+const ID = 'SearchDropDownComponent';
+const Keycodes = {
```

Comment:
Approving, but it looks like there are some keycodes in here that aren't being used. Is it possible to clean that up? This would actually be a great utility import as well (for another day, maybe)